src/test/resources/reference.conf

```diff
@@ -80,21 +80,29 @@ clusterDefaults {
   numberOfPreemptibleWorkers = 0
 }
 
-sam {
-  server = "https://sam.test.org:443"
-}
-
 swagger {
   googleClientId = "test.apps.googleusercontent.com"
   realm = "broad-dsde-test"
 }
 
 auth {
-  providerClass = "org.broadinstitute.dsde.workbench.leonardo.auth.WhitelistAuthProvider"
+  providerClass = "org.broadinstitute.dsde.workbench.leonardo.auth.SamAuthProvider"
   providerConfig = {
+    samServer = "http://127.0.0.1:1234"
+    cacheExpiryTime = 60
+    cacheMaxSize = 100
+  }
+
+  whitelistProviderConfig = {
     whitelist = ["user1@example.com"]
   }
 
+  samAuthProviderConfig = {
```

Comment:
What's the difference between `samAuthProviderConfig` and `providerConfig`? Should 1 be removed?

Reply:
`providerConfig` is the one that Leo reads in `Boot.scala`. tests often go and instantiate their own versions of `LeoService` et al using different config keys